### PR TITLE
Task/gv 988 add edit subscription module for investor

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+type mode = {
+  label: string;
+  value: DetachFromSignalProviderModeEnum;
+};
+
+const modes: { [key: string]: mode } = {
+  none: { label: "Manual closing", value: "None" },
+  closeOnly: { label: "Close only", value: "ProviderCloseOnly" },
+  closeAll: { label: "Close all immediately", value: "CloseAllImmediately" }
```
Comment (pavelzubov) [inline packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: Change labels values to translations

File: packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+          >
+            {t("unfollow-program.submit")}
+          </GVButton>
+        </div>
+      </div>
+    </form>
+  );
+};
+
+const ProgramUnfollowForm = React.memo(
```
Comment (pavelzubov) [inline packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: move React.memo inside compose

Comment (Anchello) [reply packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: Done

File: packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx (hunk 1 above)
Comment (Anchello) [reply packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: Done

File: packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { DetachFromSignalProviderModeEnum } from "gv-api-web";
+import React from "react";
+import { InjectedTranslateProps, translate } from "react-i18next";
+import { compose } from "redux";
+import GVButton from "shared/components/gv-button";
+import GVFormikField from "shared/components/gv-formik-field";
+import GVTextField from "shared/components/gv-text-field";
+import Select from "shared/components/select/select";
+
+const _ProgramUnfollowForm: React.FC<Props> = React.memo(
```
Comment (pavelzubov) [inline packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: Move React.memo **inside** compose.
```
compose<React.ComponentType<OwnProps>>(
  React.memo, 
  translate(),
```

Comment (Anchello) [reply packages/investor/src/modules/program-unfollow/components/program-unfollow-form.tsx]: Done

**Pull request merged**